toonz/sources/image/sgi/filesgi.cpp

```diff
@@ -1210,7 +1210,7 @@ void SgiWriter::writeLine(short *buffer) {
 
 Tiio::SgiWriterProperties::SgiWriterProperties()
     : m_pixelSize("Bits Per Pixel")
-    , m_endianess("Endianess")
+    , m_endianess("Endianness")
```

Comment:
This line is also used as the key term of file priperty. I think it should not be fixed, either.